README.md

```diff
@@ -65,7 +65,7 @@ Panel works really well with the visualization tools you already know and love l
 
 Panel provides bi-directional communication making it possible to react to clicks, selections, hover etc. events.
 
-[![Vega Selections](https://blog.holoviz.org/images/vega_selection.gif)](https://panel.holoviz.org/reference/panes/Vega.html)
+[![Vega Selections](https://assets.holoviz.org/panel/readme/vega_selections.gif)](https://panel.holoviz.org/reference/panes/Vega.html)
```

Comment:
The old gif. Did not exist any more. Somebody cleaned up the blog. I've uploaded another gif.

Reply:
Perfect, thanks!